judgments/feeds.py

```diff
@@ -60,8 +60,15 @@ def root_attributes(self):
 
     def add_item_elements(self, handler, item) -> None:
         super().add_item_elements(handler, item)
-        handler.addQuickElement("tna:uri", item.get("uri", ""))
         handler.addQuickElement("tna:contenthash", item.get("content_hash", ""))
+        path = item.get("uri", "")
+        handler.addQuickElement("tna:uri", path)
+        if path:
+            path_underscore = path.replace("/", "_")
+            handler.addQuickElement("tna:xml", f"/{path}/data.xml")
```

Comment:
These don't feel quite right - what does the output look like? Should we be generating `link rel="alternate"` nodes?